translations/harbour-counter-hu.ts

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="hu_HU">
```

Comment:
And here I would just say "hu" (the _HU part shouldn't be necessary) but that's again something that I can clean up myself.

Reply:
Thanks, it was setup by the Linguist, but I am not aware any other hungarian locale, so it si going to be fine.